src/cli/cli.cpp

```diff
@@ -2083,42 +2077,56 @@ void Interpreter::ProcessPing(int argc, char *argv[])
         index++;
     }
 
-    HandlePingTimer();
+    mPingIdentifier++;
 
-    return;
+    if (mPingIdentifier == 0)
+    {
+        mPingIdentifier++;
```

Comment:
Should mPingIdentifier increase for each ping caused by the same ping command?

Reply:
ICMP echo request/response has its own seq counter `icmp_seq` which is increased per message and is included in output/print message to user. 

The `mPingIndentifier` is tied to every `ping` command issued by user. Note that this is only used internally (the `id` is not printed and known to user).